components/library/RichTextEditor/index.ts

```diff
@@ -0,0 +1,6 @@
+// Index files for each component should export the tabs, preview, and title
+export { tabs } from "./tabs";
+export { Preview } from "./Preview/index";
+export const title = "Rich Text Editor";
+export const contributor = "Evan Forge"
```

Comment:
This should be your GH username, I think at some point I'll show the contributor on the component page and will use this variable to also link to the GH user.

Reply:
OK